src/components/views/visualization/GraphComponent.js

```diff
@@ -178,7 +178,7 @@ render (){
                     <div>
                       <span className="rv-hint__title"> {this.state.hoverValue.grpby_name}</span>
                       <br />
-                      <span className="rv-hint__title">Fiscal Year : </span>
+                      <span className="rv-hint__title">Month(s) : </span>
```

Comment:
months can only be shown if present in data rite ? or adding a default month ?

Reply:
I have added a default month as May for now